### DISCUSSION TITLE
Support for IVF_SQ4

### DISCUSSION THREAD

**Discussion by huberl0** [milvus-io/milvus#36312] [Ideas & Feature requests]
Hi all,

is there any plan to support IVF_SQ4 in a future release? I could not find it on the roadmap.

**Comment by yhmo** [comment]
So far we don't have any plan for IVF_SQ4.
Could you tell me why you want it? To save more memory?

**Comment by huberl0** [comment]
Yes exactly, we have a huge collection that is very expensive to store without SQ4.

**Reply by xiaofan-luan** [reply]
Using IVFPQ could achive similar memory as IVFSQ4.
In latest 2.5, HNSW + PQ will be supported as well.
